articles/support/tickets.md

```diff
@@ -19,23 +19,39 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+1. From [Support Center](${env.DOMAIN_URL_SUPPORT}), click on the **Open Ticket** button. If you don't see this button, then you do not have access to support (only paying and trial customers have access to Support). In this case you can use our [Community](https://community.auth0.com/) instead (click **Ask our Community**).
 ![Support Center](/media/articles/support/open-ticket.png)
-1. Choose the **Affected Account** from the dropdown menu.
-1.  Under **What can we help you with?** select the type of issue that best fits your case.
+1. Choose **Affected Tenant** from the dropdown menu.
+1. Under **Issue Type** select the type of issue that best fits your case.
 ![Issue types](/media/articles/support/issue-types.png)
-    * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
-        * With Service Issues, you will be asked to select a severity:
-            *  **Low: Product Question** - You have questions about how things work, whether we support something
+    * **Public Cloud Support Incident** or **Appliance Support Incident** - (the second option is only available to Appliance customers)
+        * If you are not a Trial Tenant customer, you will be asked to select a severity:
+            * **Low: Product Question** - You have questions about how things work, whether we support something
             * **Normal: General Support Question** - You are in development and have a question about how to configure something or how to resolve an error
             * **High: Production Application Issue** - Your Production application is experiencing a minor outage impacting some users/feature(s)
             * **Urgent: Production Application Offline** - Your Production application is experiencing a critical outage impacting all users
-        *  You will then be asked to choose an option under "What is the service issue related to?"
-            *  _You may be prompted for additional information depending on your selection._
-    * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
-    * **Auth0 Service Feedback** - You would like to suggested product improvements or enhancement requests
-    * **Vulnerability Reports or Legal Issues** - You would like to report security vulnerabilities or legal questions
-    * **Other** - You have a question that does not fall under any of the above categories
+        *  Depending on the issue type you chose after you select the severity, you may need to answer the **What can we help you with?**. Choose one of the following:
+            * **Public Cloud Support Incident**
```

Comment:
The options for "what can we help you with"  here are wrong. You can see the right options in following screenshot:
<img width="974" alt="whatcanwehelpyouwith" src="https://user-images.githubusercontent.com/1350396/44356104-c99dcb00-a484-11e8-8fa3-5ceefa933166.png">
But since we are not adding any description of each option I would directly remove the list of options if you agree @tamigoodall

Reply:
I agree. I'll remove the list but leave the notes below.